src/amb_kf.c

```diff
@@ -524,6 +524,8 @@ void set_nkf_matrices(nkf_t *kf, double phase_var, double code_var,
                   kf->decor_obs_mtx);
 }
 
+/* Currently this function is only used to find the index of a prn in a
+ * permuted basis, so all callsites assert the result is not -1. */
```

Comment:
Turn this into a Doxygen style comment with description of parameters and return values. There is an example of the formatting in the eng. playbook.